pyiron/thermodynamics/hessian.py

```diff
@@ -18,40 +18,115 @@
 
 class HessianJob(GenericInteractive):
     def __init__(self, project, job_name):
-        super(GenericInteractive, self).__init__(project, job_name)
+        super(HessianJob, self).__init__(project, job_name)
         self.__version__ = "0.0.1"
         self.__name__ = "HessianJob"
-        self.interactive_cache = {
-            "forces": [],
-            "positions": [],
-            "energy_pot": []
-        }
+        self.server.run_mode.interactive = True
```

Comment:
We should not enforce a given run_mode - the run_mode should be set by the user instead.